docs/website/docs/examples/_examples-header.md

```diff
@@ -0,0 +1,22 @@
+import Admonition from "@theme/Admonition";
+import CodeBlock from '@theme/CodeBlock';
+
+<Admonition>
+    The <span>{props.title}</span> Example is part of the <a href="/docs/examples">DLT Code Examples</a>, a list of comprehensive examples to help you solve real world loading problems with dlt.
+    The source code for this example can be found in our repository at: <a href={"https://github.com/dlt-hub/dlt/tree/devel/docs/examples/" + props.slug}>{"https://github.com/dlt-hub/dlt/tree/devel/docs/examples/" + props.slug}.</a>
+</Admonition>
+
+## TLDR
+<div>{props.intro}</div>
+
+## Setup: Running this example on your machine
+<CodeBlock language="sh">
```

Comment:
any reason to use this and not
```
```sh
```

Reply:
Yes, the variable will not work in the markdown block unfortunately..